src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -193,6 +197,18 @@ private static string GetWinFormsLoaderName(IVsHierarchy vsHierarchy)
                 if (frameworkName.Identifier == ".NETCoreApp" &&
                     frameworkName.Version?.Major >= 3)
                 {
+                    if (!(_oleServiceProvider.QueryService<SVsShell>() is IVsShell shell))
+                    {
+                        return null;
+                    }
+
+                    var newWinFormsDesignerPackage = new Guid("c78ca057-cc29-421f-ad6d-3b0943debdfc");
+                    if (!ErrorHandler.Succeeded(shell.IsPackageInstalled(newWinFormsDesignerPackage, out var installed))
+                        || installed != 1)
```

Comment:
```suggestion
                        || installed == 0)
```

(not all COM trues are 1 so I'm just paranoid)

Reply:
Amended the original commit to include this change.